drop-boxes/register-omero-metadata/image_registration_process.py

```diff
@@ -73,6 +101,10 @@ def requestOmeroDatasetId(self, project_code=None, sample_code=None):
         return ds_id
 
     def registerImageFileInOmero(self, file_path, dataset_id):
+        print "using file_path:"
+        print file_path
+        print "ds id:"
+        print dataset_id
```

Comment:
Seeing this in the logs is not very informative. Can you make the logging more expressive please?

Reply:
removed those lines, we already print that on register-omero.py